src/Notifier.cpp

```diff
@@ -144,17 +164,17 @@ size_t Notifier::Notify(const std::vector<std::string>& messages) {
 size_t Notifier::ReceiveConfirmation() {
    StopWatch waitCheck;
    size_t responses = 0;
-   while (responses < gHandshakeCount && waitCheck.ElapsedSec() < gMaxTimeoutInSec) {
+   while (responses < gHandshakeCount && waitCheck.ElapsedSec() < mMaxTimeoutInSec) {
       std::string msg;
-      if (gHandshakeQueue->GetShot(msg, gMaxTimeoutInSec)) {
+      if (mHandshakeQueue->GetShot(msg, mMaxTimeoutInSec)) {
          LOG(INFO) << "Received update confirmation from thread #"
```

Comment:
Maybe this should be a LOG(DEBUG). Is there a good reason to have this in the production build?